src/nanoemoji/nanoemoji.py

```diff
@@ -209,7 +209,7 @@ def _chrome_command() -> str:
 
 def write_config_preamble(nw, font_config: FontConfig):
     if font_config.has_bitmaps:
-        # TODO bitmap support for non-square svgs? - baby steps
+        # conveniently if w != h this already scales w
```

Comment:
wait, what does that mean exactly?

When w != h, what we want from resvg is that the resulting bitmap always has its height set to the desired `font_config.bitmap_resolution` (e.g. 128) whereas the bitmap width can be variable and scaled accordingly.
Remember the relative "size" of our glyhps is determined by the viewBox height which is scaled to fit the sum of ascender - descender.

I just tried to experiment passing resvg some SVGs with viewbox w != h and passing -w and -h to the same value (like you are doing here), and it looks like resvg is just scale-to-fitting to the maximum of either width or height.
We don't want that actually. If a proportional glyph is wider than tall, we want the bitmap height set to the bitmap resolution and the bitmap width set wider relative to its aspect ratio; we don't want the bitmap width set to the resolution and the height scaled down.

Actually I tried to compile a proportional cbdt font from a couple of SVGs, where one has a viewbox width that is larger than its height (e.g. one has viewBox="0 0 10 10" and the other "0 0 20 10"), and I got this error (seems like you only tested with "narrow" proportional glyphs) 

```
  File "/Users/clupo/Github/nanoemoji/src/nanoemoji/bitmap_tables.py", line 213, in _make_cbdt_strike
    bitmap_pixel_height = only({c.bitmap.size[1] for c in color_glyphs})
  File "/Users/clupo/Github/nanoemoji/src/nanoemoji/util.py", line 34, in only
    assert next(it, None) is None
AssertionError
ninja: build stopped: subcommand failed.
```

Reply:
I believe what we want is fix only the height to resvg -h, and let resvg adjust the bitmap width proportionately, i.e. `f"resvg -h {res} $in $out"`.

Reply:
argh.. if I fix the height and let the width free to expand, I reach a point where the latter can't fit any longer in the ridiculously tight uint8 limits of CBDT SmallGlyphMetrics!

https://docs.microsoft.com/en-us/typography/opentype/spec/cbdt#smallglyphmetrics

Seriously, does it mean one can't have a PNG in CBDT table if height or width exceed 255?
Wow..

Reply:
> I believe what we want is fix only the height to resvg -h, and let resvg adjust the bitmap width proportionately, i.e. f"resvg -h {res} $in $out".

Quite right ty, I definitely only tested narrow so far. Will update.

> does it mean one can't have a PNG in CBDT table if height or width exceed 255

I believe so, yes. If we agreed on which way y goes we'd use sbix but ... no such luck.